source/_docs/locked_out.md

```diff
@@ -20,7 +20,24 @@ Connect a keyboard and monitor to your device.
 
 `auth reset --username existing_user --password new_password`
 
-### Home Assistant Core and Home Assistant Container
+### Home Assistant Container
+
+If you are running Home Assistant in a container you can login into the container and use the `hass` command line tool to change your password. The following example shows how to do it using `docker-compose`, you can adapt the below commands if using `docker` only. The container running Home Assistant is called `ha` in the below example:
+
+```bash
+# Log into the container
+docker-compose exec ha bash
```

Comment:
Our documentation is not using Docker compose, so it is weird to have a Docker compose reference in here all of a sudden.

Besides, this can be done without Docker compose, in a more universal way, right?

Reply:
The doc specifies how to change the password when running ha in a container. I can put the `hass` command only if you prefer but this complete example is easy to understand and people can also just pick the `hass` command and adapt it to their needs.

Reply:
Please ping again if you want me to remove `docker-compose` totally.

Reply:
Alternatively, one can docker exec on the container directly. That way, it doesn't matter how it was set up.

Reply:
That's what I meant by "you can adapt the below commands if using `docker` only".

Reply:
> "you can adapt the below commands if using docker only".

Sorry, you make no sense at this point. There are no docker commands below.

Your PR adds commands for recovery when using docker-compose. However, the Home Assistant project currently does not use that approach in general. Besides, running without compose is a perfectly valid way of running in Docker as well.

So my proposal is to change this part into a direct docker exec, e.g., 

`docker exec homeassistant --script auth --config /config ...`

That will work on any system, regardless if one uses composer or not.